lib/Runtime/Library/DateImplementation.cpp

```diff
@@ -1235,6 +1237,12 @@ namespace Js {
                             goto LError;
                         }
                         lwMonth = pszs->lwVal;
+                        if ('-' == *pch)
```

Comment:
can we have a totally bogus test of `-` is being the last character? i.e. `Wed, 11 Oct-"